src/tbp/monty/frameworks/config_utils/make_dataset_configs.py

```diff
@@ -509,47 +514,51 @@ def get_omniglot_eval_dataloader(
 
 @dataclass
 class SensorAgentMapping:
-    agent_ids: List[str]
+    agent_ids: list[AgentID]
     sensor_ids: List[str]
-    sensor_to_agent: Dict
+    sensor_to_agent: dict[str, AgentID]
```

Comment:
It might be out of scope, but do these `str` keys represent something that we might want a newtype for? It might be part of a larger refactor though, so I'm fine with leaving it off for now.